Stop using .container-fluid and .breadcrumb bootstrap classes

KKoukiou opened the pull request.

KKoukiou commented:
@marusak @martinpitt can any of you review this one? It's an easy one

marusak commented:
I'll take this one.

marusak commented:
Looking at the UI first.
- in graphs there are two issues - There is no `>` between `Overview` and the subsequent item, and the other `>` is not spaced properly.
![broken](https://user-images.githubusercontent.com/12330670/91850978-b8d0e400-ec5e-11ea-9a9a-da4e5ec61be0.png)

- it feels misaligned on services page: (the same with applications, hardware details, virtual machines)
![servicesmissaligned](https://user-images.githubusercontent.com/12330670/91851192-0b120500-ec5f-11ea-8b3e-27474889357b.png)
(at least when compared to logs (networking, storage...) (plus the `>` is bold on those pages)
![logscompared](https://user-images.githubusercontent.com/12330670/91851280-2d0b8780-ec5f-11ea-9650-81fdf483b044.png)

- the spacing on accounts feels wrong (at least not consistent with the rest of the cockpit)
![acountspacing](https://user-images.githubusercontent.com/12330670/91851345-4dd3dd00-ec5f-11ea-8ff1-ab9959158658.png)

KKoukiou commented:
@marusak I think both issues you mentioned are issues in master and are not caused by this PR.

KKoukiou commented:
I will address the issues you mentioned and repost.

KKoukiou commented:
@marusak I fixed both issues you found in another PR, which is already merged, and rebased. https://github.com/cockpit-project/cockpit/pull/14550

Can you please re-review?

marusak reviewed (changes requested):
One regression unfortunately.

KKoukiou commented on pkg/kdump/kdump-view.jsx:
```diff
@@ -21,7 +21,7 @@ import cockpit from "cockpit";
 import React from "react";
 import { OnOffSwitch } from "cockpit-components-onoff.jsx";
-import { Button, Tooltip, TooltipPosition } from "@patternfly/react-core";
+import { Button, Tooltip, TooltipPosition, Page, PageSection, PageSectionVariants } from "@patternfly/react-core";
```
Good catch, reposting.

KKoukiou replied:
btw I just noticed that this page is broken on mobile, even on master, it does not wrap at all. I will send another PR for fixing this.

marusak reviewed (approved):
Awesome, thanks!
